crates/BUILD

```diff
@@ -0,0 +1,76 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+exports_files(["rules.bzl"])
+
+load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_jvm_library")
+
```

Comment:
some of the target have their visibility set to `public`, however they're mostly referred from `rules.bzl`, which exists in the same package. therefore should we set them all to `private` instead?

Reply:
@lolski they need to be visible from `deploy_crate` target, not from `rules.bzl`; marking them as private would fail the build with this error:

>ERROR: in assemble_crate rule //grammar:assemble-crate: target '@vaticle_bazel_distribution//crates:crate-assembler' is not visible from target '//grammar:assemble-crate'. Check the visibility declaration of the former target if you think the dependency is legitimate